ledger-service/http-json/src/main/scala/com/digitalasset/http/util/ErrorOps.scala

```diff
@@ -22,4 +22,13 @@ object ErrorOps {
     def liftErrS[M](msg: String)(f: String => M)(implicit L: Show[L]): M \/ R =
       liftErr(x => f(msg + " " + x))
   }
+
+  implicit final class `EitherT WSS extras`[F[_]: Functor, L, R](private val self: EitherT[F, L, R])
```

Comment:
IDK what WSS means but I just took that name from above :joy:

Reply:
I believe it's "web socket service".